x-pack/plugins/search_playground/public/components/summarization_panel/open_ai_summarization_model.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiIcon,
+  EuiSuperSelect,
+  EuiSuperSelectOption,
+  EuiToolTip,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { SummarizationModelName } from '../../types';
+
+import { OpenAIIcon } from './open_ai_icon';
+
+const renderSelectOptions = (label: string) => (
+  <EuiFlexGroup alignItems="center">
+    <EuiFlexItem grow={false}>
+      <EuiIcon type={OpenAIIcon} />
+    </EuiFlexItem>
+    <EuiFlexItem>{label}</EuiFlexItem>
+  </EuiFlexGroup>
+);
+
+const SummarizationModel: Array<EuiSuperSelectOption<string>> = [
```

Comment:
do we need to hard-code these? this seems pretty brittle given the rate of change?

Is there a way for a user to manually input a model as well?

Reply:
New ones are added fairly frequently but not removed. For now lets keep as is and I will create a ticket to consider the options of:
- pulling the list of LLMs externally
- Using the Kibana LLM connectors to manage the chosen / available models